pkg/collector/corechecks/oracle-dbm/sysmetrics.go

```diff
@@ -125,14 +125,16 @@ func (c *Check) sendMetric(s sender.Sender, r SysmetricsRowDB, seen map[string]b
 			log.Debugf("%s %s: %f", c.logPrompt, metric.DDmetric, value)
 			s.Gauge(fmt.Sprintf("%s.%s", common.IntegrationName, metric.DDmetric), value, "", appendPDBTag(c.tags, r.PdbName))
 			seen[r.MetricName] = true
+			*n++
 		}
 	}
 }
 
-func (c *Check) SysMetrics() error {
+func (c *Check) SysMetrics() (int64, error) {
+	var n int64
```

Comment:
What's the use of `n`? is it an internal counter?

Reply:
Yes, it's an internal counter of the emitted sys metrics. I exposed it in the return value for tests.

Reply:
ok. can we add a docstr to this function? it was not very clear to me that the counter is only used for tests, especially we ignored the counter [here](https://github.com/DataDog/datadog-agent/pull/21031/files#diff-c4f29b606f87d4700c343a33d1ebc82e5e85fbeb8697c61034c307056879678dR200).

Reply:
done